docs/book/src/reference/metrics.md

```diff
@@ -69,6 +69,61 @@ NAME                                         AGE
 monitor-controller-manager-metrics-monitor   2m8s
 ```
 
+### RBAC for Prometheus
+
+When using [prometheus-operator](https://github.com/prometheus-operator) for
+cluster monitoring, you should ensure that necessary RBAC roles and permissions
+were applied to the namespace where the manager is deployed. Otherwise,
+Prometheus server will not be able to access the ServiceMonitor that describes
+the set of targets to be monitored by Prometheus.
+
+```yaml
```

Comment:
Do we need to add it here? is not that in the Prometheus Operator doc? 
Is not better only so that if they cange that we do not get this doc outdated?